resources/fetchers/elb_provider.go

```diff
@@ -29,8 +29,7 @@ func (provider ELBProvider) DescribeLoadBalancer(ctx context.Context, balancersN
 	req := provider.client.DescribeLoadBalancersRequest(input)
 	response, err := req.Send(ctx)
 	if err != nil {
-		logp.Err("Failed to describe load balancers %s from elb, error - %+v", balancersNames, err)
```

Comment:
I see you have deleted many error logs, why is that?

Reply:
I just want the beats to be aligned.

No need to log an error and then propagate it back to the caller and then log again.